invenio_app_ils/config.py

```diff
@@ -1071,3 +1078,6 @@ def _(x):
 ILS_PATRON_SYSTEM_AGENT_CLASS = SystemAgent
 
 DB_VERSIONING_USER_MODEL = None
+
+# Feature Toggles
+SELF_CHECKOUT_ENABLED = False
```

Comment:
to keep the config variables naming consistent:
```suggestion
ILS_SELF_CHECKOUT_ENABLED = False
```